vpr/src/draw/draw_types.h

```diff
@@ -163,6 +163,9 @@ typedef struct {
  * save_graphics_file_base: Base of save graphis file name (i.e. before extension)
  * pres_fac: present congestion cost factor
  */
+/* Note: t_draw_struct is used in the same way as a Context, but cannot be a Context because Contexts are
+ * not copyable, while t_draw_struct must be.
```

Comment:
Say why it must be copied; graphics commands feature uses that to do save an restore of the drawing state.